### ISSUE TITLE
Hyper modifier is not recognised?

### ISSUE THREAD

**Issue by krompus** [carnager/clerk#18]
I would like to bind some controls to the Hyper key. I have remapped my keyboard to have one, and I use Hyper keybinds outside of clerk to send `mpc` commands (highly recommended!)

I've tried mapping a few random commands to Hyper keybinds, and clerk starts without error (indeed, the error screen says that Hyper is a valid key), and [it shows up in the command hints.](https://ptpb.pw/togp.png)

"Hyper+r: Add + play"

which I specified with `addplay="Hyper+x"` in `~/.config/clerk/config`

When I try these keybinds, though, it just types the letters in the search field as if I weren't holding the Hyper modifier.

I have the key mapped as `Hyper_L` (keysym 0xffed), and I tried remapping it to be `Hyper_R` (keysym 0xffee), but I get the same result.

I also tried `addplay="Hyper_L+r"` and `addplay="Hyper_R+r"` with each of the different key mappings, and none of the configurations work.

Is this a bug, or do I need to change something in my configuration?

**Comment by carnager** [comment]
Just checked rofi documentation. It seems the key is called "HyperL"

**Comment by krompus** [comment]
Herm. I've replaced some values in clerk's config:

    toggle="HyperL+Space"
    prev="HyperL+comma"
    next="HyperL+period"

These all work with Control, but not with Hyper.

I replaced one of the commands in rofi's config:

    rofi.kb-move-end:  HyperL+e

(it was `Control+e` before)

Now, `Control+e` does nothing (confirming that it's recognises that I've remapped the command), but `HyperL+e` enters `e` as if I didn't have the Hyper key pressed.

It seems this is a `rofi` bug then. I'll go bug them. :)

Thanks!
